app/test/unit/email-test.ts

```diff
@@ -259,38 +271,44 @@ describe('emails', () => {
         'free'
       )
 
-      expect(isAttributableEmailFor(account, 'personal@gmail.com')).toBeTrue()
-      expect(isAttributableEmailFor(account, 'company@github.com')).toBeTrue()
-      expect(
-        isAttributableEmailFor(account, 'niik@users.noreply.github.com')
-      ).toBeTrue()
-      expect(
-        isAttributableEmailFor(account, '123+niik@users.noreply.github.com')
-      ).toBeTrue()
+      assert.equal(isAttributableEmailFor(account, 'personal@gmail.com'), true)
```

Comment:
Same with the lines below, right?
```suggestion
      assert(isAttributableEmailFor(account, 'personal@gmail.com'))
```